src/main/java/com/microsoft/azure/vmagent/AzureVMAgentCleanUpTask.java

```diff
@@ -96,6 +105,7 @@ void decrementAttemptsRemaining() {
         private String cloudName;
         private String deploymentName;
         private String resourceGroupName;
+        private String scriptUri;
```

Comment:
Will this be null when deserializing from old data?

Reply:
The previous version doesn't serialize the `deploymentInfo`